spec/form_models/job_summary_form_spec.rb

```diff
@@ -19,17 +19,37 @@
     end
 
     describe '#about_school' do
-      job_summary_form = JobSummaryForm.new(about_school: nil)
+      let(:job_summary_form) { JobSummaryForm.new(about_school: nil, job_location: job_location) }
 
       context 'when about school is blank' do
-        let(:about_school) { nil }
+        context 'for a vacancy at a single school in a trust' do
+          let(:vacancy) { create(:vacancy, :with_school_group_at_school) }
+          let(:job_location) { 'at_one_school' }
 
-        it 'requests an entry in the field' do
-          expect(job_summary_form.valid?).to be false
-          expect(job_summary_form.errors.messages[:about_school])
-            .to include(
-              I18n.t('activemodel.errors.models.job_summary_form.attributes.about_school.blank')
-            )
+          it 'requests the user to complete the about school field' do
+            expect(job_summary_form.valid?).to be false
+            expect(job_summary_form.errors.messages[:about_school].first).to match(/school/)
+          end
+        end
+
+        context 'for a vacancy at a single school NOT in a trust' do
+          let(:school) { create(:school) }
+          let(:job_location) { nil }
+
+          it 'requests the user to complete the about school field' do
+            expect(job_summary_form.valid?).to be false
+            expect(job_summary_form.errors.messages[:about_school].first).to match(/school/)
+          end
+        end
+
+        context 'for a vacancy at a school group' do
+          let(:vacancy) { create(:vacancy, :with_school_group) }
```

Comment:
I know I created most of these but I think we aren't using any of these `let(:vacancy)`s or `let(:school)`s: they are never referenced in the test examples.